src/applications/vaos/new-appointment/redux/actions.js

```diff
@@ -659,6 +659,12 @@ export function checkCommunityCareEligibility() {
           isEligible: response.eligible,
         });
 
+        if (response.eligible) {
+          recordEvent({
+            event: `${GA_PREFIX}-cc-eligible-yes`,
```

Comment:
@grossb1 We might need to verify this event name with the analytics folks, if you haven't.

Reply:
@jbalboni I reached out to @lenaecb this morning to verify the name, should I be running it by others as well?

Reply:
If she verified it, that's good enough, I think.